admin/controllers/tags/views/edit/model.php

```diff
@@ -36,7 +36,7 @@
 
 	// validate
 	if ($required_details_form->validate()) {
-		if (!$custom_fields_form || ($custom_fields_form && $custom_fields_form->validate()) ) {
+		if (!$custom_fields_form->id || ($custom_fields_form && $custom_fields_form->validate()) ) {
```

Comment:
Does this addition of ->id need to be null coalesced to avoid error if custom_fields_form doesn't have the property?